lib/deployinator/helpers.rb

```diff
@@ -592,12 +592,26 @@ def average_duration(type, stack)
       avg_time
     end
 
+    # options:
```

Comment:
Thanks for adding the option documentation, could you also add a quick description of what the function does directly above the options descriptions?